Dockerfile

```diff
@@ -48,28 +43,16 @@ RUN dpkg-reconfigure locales
 
 RUN useradd -m onadata
 RUN mkdir -p /srv/onadata && chown -R onadata:onadata /srv/onadata
-USER onadata
-RUN mkdir -p /srv/onadata/requirements
 
-ADD requirements /srv/onadata/requirements/
+USER onadata
 
-WORKDIR /srv/onadata
+ADD . /srv/onadata
```

Comment:
Codacy found an issue: [Use COPY instead of ADD for files and folders](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6162537)